### PR TITLE
Update french titles and descriptions

### PR DESCRIPTION
**Pull request by ath0mas**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

Fix some wording for titles and descriptions in french locale.

### CODE REVIEW

**Review by maniaxxx — APPROVED** [review]

File: _articles/fr/best-practices.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 lang: fr
-title: Bonnes Pratiques pour les Responsables
-description: Facilitez vous la vie en tant que responsable open source, de la documentation des processus à l'exploitation de votre communauté.
+title: Bonnes pratiques pour les responsables
```
Comment (gallexi) [inline _articles/fr/best-practices.md:3]:
```suggestion
title: Bonnes Pratiques pour les Responsables
```

File: _articles/fr/code-of-conduct.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 lang: fr
-title: Votre Code de Conduite
+title: Votre code de conduite
```
Comment (gallexi) [inline _articles/fr/code-of-conduct.md:3]:
```suggestion
title: Votre Code de Conduite
```

File: _articles/fr/leadership-and-governance.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 lang: fr
-title: Leadership et Gouvernance
-description: Les projets open source en croissance peuvent bénéficier de règles formelles pour prendre des décisions.
+title: Leadership et gouvernance
```
Comment (gallexi) [inline _articles/fr/leadership-and-governance.md:3]:
```suggestion
title: Leadership et Gouvernance
```

File: _articles/fr/legal.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 lang: fr
 title: Le côté légal de l'Open Source
```
Comment (gallexi) [inline _articles/fr/legal.md:3]:
```suggestion
title: Le Côté Légal de l'Open Source
```

File: _articles/fr/starting-a-project.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 lang: fr
 title: Lancer un projet Open Source
```
Comment (gallexi) [inline _articles/fr/starting-a-project.md:3]:
```suggestion
title: Lancer un Projet Open Source
```

**Review by gallexi — COMMENTED** [review]
Ça m’a l’air génial, merci pour votre contribution!

What do you think about undoing the title case changes (and updating some others to match)? I've made some suggested changes there. Otherwise, this looks great!

**Review by Rithy77 — COMMENTED** [review]
rithy2505@gmail.com

**Comment by ath0mas** [comment]
> 
> 
> Ça m’a l’air génial, merci pour votre contribution!
> 
> What do you think about undoing the title case changes (and updating some others to match)? I've made some suggested changes there. Otherwise, this looks great!

Sorry but it is considered bad practice to capitalize words even in titles in French ; it is limited to few [rules](https://fr.wikipedia.org/wiki/Usage_des_majuscules_en_fran%C3%A7ais) only.

**Review by gallexi — APPROVED** [review]
Fair enough! Thanks for your contribution!

**Pull request merged**